components/threads/threads.app.mjs

```diff
@@ -32,24 +32,25 @@ export default {
     blocks: {
       label: "Blocks",
       type: "string[]",
-      description: "Add one or more blocks to your thread. You can use [Markdown](https://www.markdownguide.org/basic-syntax/) to format text. Try `# This is a block`",
+      description:
+        "Add one or more blocks to your thread. You can use [Markdown](https://www.markdownguide.org/basic-syntax/) to format text. Try `# This is a block`",
```

Comment:
```suggestion
        "Add one or more blocks to a thread. You can use [Markdown](https://www.markdownguide.org/basic-syntax/) to format text. Try `# This is a block`"
```

Reply:
Changed "your" to "a" to follow Google's style guide on addressing the user.

Reply:
Did you want me to remove the trailing `,`? It seems that they're everywhere in this document and would introduce a style mismatch.

This might be a better task to prettier the entire repository with something automated, after the fact?